qlib/kernel/Kernel.rs

```diff
@@ -13,21 +13,40 @@
 // limitations under the License.
 
 use super::super::common::*;
+#[cfg(not(feature = "cc"))]
```

Comment:
well, this is pretty noisy, perhaps you could simply just import them, the current config won't warn unused imports anyways

Reply:
It will warn unused imports...